bundles/org.openhab.io.homekit/README.md

```diff
@@ -724,6 +724,21 @@ All enum values can be customized via item metadata. I.e. `HEAT="heating", COOL=
 They are appropriately marked.
 Enums that are linked to Switches or Contacts have an `inverted` param that will reverse the sense of `ON`/`OFF` or `OPEN`/`CLOSED`.
 
+Enum mappings can have multiple values for a single key.
+These must be an array, not a comma separated string.
```

Comment:
Filie based configuration should be fully supported. It should be possible to have a comma seperated list in the config. Both file based and UI based config should be able to handle that. 
For reference you can look at [the tr064 binding](https://www.openhab.org/addons/bindings/tr064/#a-note-on-textual-configuration) => `macOnline` property.

Reply:
I'm [working on it](https://github.com/openhab/openhab-core/pull/4330). Thanks for the reference though, it helps me figure out this xtext thing.